### ISSUE TITLE
Weird behaviour when inputting nothing

### ISSUE THREAD

**Issue by badboy** [jamesmunns/frauth#23]
During `frauth init` the user has to enter a few things.

If the prompt is displayed (e.g. "Comment:") and I only press Enter, the line stays unchanged or erases the previously printed lines (the output of frauth itself).

It should be possible to abort input/allow empty input.
In case of empty input the tool could still ask whether that was intentional.

**Comment by jamesmunns** [comment]
Makes sense! Not sure what is up with the prompt issues, I'll try to reproduce locally. You're on Mac right? Any particular shell or terminal emulator?

**Comment by badboy** [comment]
zsh on iTerm2.
